src/pages/Hojung/Main/Main.js

```diff
@@ -1,8 +1,27 @@
 import React from 'react';
+import './Main.scss';
+import MainFeed from './MainFeed';
+import SideMenu from './SideMenu';
 
 class MainHojung extends React.Component {
+  logout = () => {
+    localStorage.removeItem('token');
+    this.props.history.push('/login-hojung');
+  };
   render() {
-    return <div>hi</div>;
+    const username = this.props.history.location.state.username;
+    console.log(username);
+    return (
+      <>
+        <div id="main">
+          <div className="contain">
```

Comment:
```suggestion
        <div className="main">
          <div className="contain">
```
- 불필요한 프래그먼트가 있습니다.
- 지금 보니 contain style이 main 이하에 있는데 css 파일에서 nesting 해줘야 할거 같습니다.

Reply:
수정완료 했습니다